### ISSUE TITLE
Update screenshots and documentation for the latest version

### ISSUE THREAD

**Issue by D4rK7355608** [heinrichreimer/android-issue-reporter#158]
### Is your feature request related to a problem? Please describe.
The screenshots and documentation for the android-issue-reporter library on the GitHub page and the Google Play Store demo app are outdated. This could lead to confusion or misinterpretation of the library's current state and functionality.

### Describe the solution you'd like
I suggest updating the screenshots and documentation to reflect the latest changes and features of the library. This includes showcasing the new material design theme

### Describe alternatives you've considered
An alternative could be to maintain a changelog or version history that details the changes and updates in each version of the library. This would complement the updated screenshots and documentation, providing users with a comprehensive overview of the library's evolution.

### Additional context
Updating the screenshots and documentation would not only make it easier for users to understand how to use the library, but it would also help attract more attention and feedback to the project. Thank you for considering this suggestion.

**Comment by heinrichreimer** [comment]
Sounds good! @D4rK7355608 Would you have some time left to update the screenshots in the readme?
I could then just use the same screenshots for Google Play once we have some of the pending updates ready, and also deploy an updated APK.

**Comment by D4rK7355608** [comment]
Hi, thank you for your quick response. I'm currently working on a project where I'm facing an issue related to the android-issue-reporter library. The problem is that it's finding the version 1.3.1 and not the version 1.4.1. The error I'm getting is "Could not find com.heinrichreimersoftware:android-issue-reporter:1.4.1". Once I figure out how to fix this error, I'll definitely update the screenshots in the readme as per your suggestion.

**Comment by heinrichreimer** [comment]
Yeah, there was a build issue with the latest release :see_no_evil: 
Could you try out if `com.github.heinrichreimer:android-issue-reporter:bfdf7ac2d6` works?

**Comment by D4rK7355608** [comment]
Hi, sorry for the late notice, I was very busy this time. Sadly the `com.github.heinrichreimer:android-issue-reporter:bfdf7ac2d6` doesn't works. I saw that you made some new commits, I will take an eye. :D
